### PR TITLE
Add EU 2022 summit details

### PR DESCRIPTION
**Pull request by mrbobbytables**
/hold
for review

### CODE REVIEW

File: content/en/events/kcseu/_index.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+weight: 1
+description: >
+  Kubernetes Contributor Summit Europe, hosted in Valencia Spain.
+
+---
+
+
+### About
+
+The Europe Summit will be an lightweight event taking place on May 16th
```
Comment (tokt) [inline content/en/events/kcseu/_index.md:14]: "a lightweight" perhaps?

File: content/en/events/kcseu/_index.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+[location]: /events/kcseu/location/
+[RSVP]: /events/kcseu/registration/
+
+
+
+### Code of Conduct
+
+Attendees agree to abide by Kubernetes [Code of Conduct]. More information on
+how to report an event will be posted in the future.
```
Comment (tokt) [inline content/en/events/kcseu/_index.md:38]: Would it make sense to replace it with "a violation or suspected violation"?

File: content/en/events/kcseu/faq.md [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+
+[KubeCon Europe]: https://events.linuxfoundation.org/kubecon-cloudnativecon-europe/
+
+### Where is the summit located?
+
+The in-person location at the
+<a href="https://www.feriavalencia.com/en/" rel="noopener noreferrer" target="_blank">Feria Valencia</a>,
+See the [Location & Venue] section for more information.
+
+[Location & Venue]: /events/past-events/2021/kcsna/location/
```
Comment (tokt) [inline content/en/events/kcseu/faq.md:29]: should this be replaced with "/events/kcseu/location/" ?

File: content/en/events/kcseu/faq.md [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+
+
+
+### Will there be a New Contributor Workshop?
+
+No, with uncertain attendance and staffing, we made this decision to ensure we
+can put on an event to the best of our ability. For those that are interested
+in becoming a contributor, SIG ContribEx plans to launch a course later this year.
+
+Until then, here are you some additional resources you may find useful:
```
Comment (tokt) [inline content/en/events/kcseu/faq.md:81]: extra "you" typo.

**Review by tokt — COMMENTED** [review]
Just a few minor typos.

File: content/en/events/kcseu/faq.md [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+### Where is the summit located?
+
+The in-person location at the
+<a href="https://www.feriavalencia.com/en/" rel="noopener noreferrer" target="_blank">Feria Valencia</a>,
+See the [Location & Venue] section for more information.
+
+[Location & Venue]: /events/past-events/2021/kcsna/location/
+
+
+### Will I be able to attend remotely?
```
Comment (jberkus) [inline content/en/events/kcseu/faq.md:32]: Is this information confirmed with CNCF Staff?

File: content/en/events/kcseu/faq.md [hunk 6]
```diff
@@ -0,0 +1,91 @@
...
+- [What is a sponsored attendee?](#what-is-a-sponsored-attendee)
+- [I am not an Org member. Can I still attend?](#i-am-not-an-org-member-can-i-still-attend)
+- [Will there be a New Contributor Workshop?](#will-there-be-a-new-contributor-workshop)
+- [I have a question! How do I contact the event staff?](#i-have-a-question-how-do-i-contact-the-event-staff)
+
+### When is the summit taking place?
+
+May 16th, the Monday of [KubeCon Europe].
+
+[KubeCon Europe]: https://events.linuxfoundation.org/kubecon-cloudnativecon-europe/
```
Comment (jberkus) [inline content/en/events/kcseu/faq.md:21]: can we move these links to the end of the doc, per style guidelines?

File: content/en/events/kcseu/faq.md [hunk 7]
```diff
@@ -0,0 +1,91 @@
...
+
+Kubernetes Orgs:
+- [kubernetes](https://github.com/kubernetes)
+- [kubernetes-client](https://github.com/kubernetes-client)
+- [kubernetes-csi](https://github.com/kubernetes-csi)
+- [kubernetes-sigs](https://github.com/kubernetes-sigs)
+
+If you are not yet a member, but active within the project, please consider
+joining! The process of joining is straight forward. See our [org membership]
+docs for more information.
```
Comment (jberkus) [inline content/en/events/kcseu/faq.md:61]:
```suggestion
docs for more information.  If your application to join is pending, please [contact us] about registering.
```
[contact us]: mailto:community@kubernetes.io

File: content/en/events/kcseu/faq.md (hunk 5 above)
Comment (jberkus) [reply content/en/events/kcseu/faq.md:32]: It is not.  Replace with "We will not be able to support remote access"

File: content/en/events/kcseu/_index.md (hunk 2 above)
Comment (jberkus) [reply content/en/events/kcseu/_index.md:38]: We should already know this.  Let's link to the main kubecon site.

File: content/en/events/kcseu/faq.md (hunk 5 above)
Comment (jberkus) [reply content/en/events/kcseu/faq.md:32]: Better wording: "Based on our experience in Kubecon NA 2021, we do not expect to be able to support remote access for this event."

**Review by jberkus — CHANGES_REQUESTED** [review]

File: content/en/events/kcseu/faq.md (hunk 6 above)
Comment (mrbobbytables) [reply content/en/events/kcseu/faq.md:21]: I leave them in their "logical area" unless they are used more than once in which case they goto the bottom.

File: content/en/events/kcseu/registration.md [hunk 8]
```diff
@@ -0,0 +1,25 @@
...
+joining! The process of joining is straight forward. See our [org membership]
+docs for more information. If your application to join is pending, please
+contact us at community@kubernetes.io about registering.
+
+
+
+**NOTE:** The summit adheres to KubeCon’s [Health & Safety Guidelines] requiring
+**ALL** in-person attendees to be fully vaccinated against the COVID-19 virus.
+
+[Kubernets orgs]: /events/kcseu/faq/#why-do-i-need-to-be-a-kubernetes-org-member-to-attend-in-person
```
Comment (jberkus) [inline content/en/events/kcseu/registration.md:23]: should be "kubernetes orgs"

File: content/en/events/kcseu/faq.md [hunk 9]
```diff
@@ -0,0 +1,94 @@
...
+docs for more information. If your application to join is pending, please
+contact us at community@kubernetes.io about registering.
+
+[org membership]: https://github.com/kubernetes/community/blob/master/community-membership.md#member
+
+
+### What is a sponsored attendee?
+
+If you are a SIG lead or subproject owner and there is someone that is not yet
+an org member or from another project that might take part of a panel,
```
Comment (jberkus) [inline content/en/events/kcseu/faq.md:72]:
```suggestion
an org member or from another project that might take part in a panel,
```

**Review by jberkus — CHANGES_REQUESTED** [review]
Sorry, just two stupid minor typos.

**Review by jberkus — APPROVED** [review]

**Pull request merged**